fingerprint-dialog-compat/build.gradle

```diff
@@ -10,8 +10,7 @@ apply plugin: 'com.android.library'
 apply plugin: 'com.github.dcendents.android-maven'
 
 android {
-    compileSdkVersion 'android-P'
-    buildToolsVersion '28.0.0 rc1'
+    compileSdkVersion 28
```

Comment:
We should add `buildToolsVersion '28.0.0'` here.

Reply:
added